### DISCUSSION TITLE
Versions related

### DISCUSSION THREAD

**Discussion by makattar** [elsa-workflows/elsa-core#1251] [Q&A] [answered]
Is Elsa 2.1 also known as Elsa 2.0.0.159?
If not how can I install Elsa 2.1?

**Comment by sfmskywalker** [comment, marked as answer]
Not exactly.

Elsa 2.0 is available as Elsa.* packages with version `2.0.0.159` (the last segment 159 represents the build number).

Elsa 2.1 is available as Elsa.* packages with version `2.1.0.{x}` from [the MyGet feed](https://www.myget.org/F/elsa-2/api/v3/index.json).

Checkout [this page](https://elsa-workflows.github.io/elsa-core/docs/next/installation/installing-feeds) for instructions on how to get these packages easily by creating a Nuget.config file in the root of your project.
